muesli/web/viewsExam.py

```diff
@@ -168,10 +169,19 @@ def __call__(self):
                     exerciseStudent.exercise = e
                     points[student.student_id][e.id] = exerciseStudent
                     self.db.add(exerciseStudent)
+
+        db_admissions = exam.exam_admissions.filter(ExamAdmission.student_id.in_([s.student.id for s  in students])).all()
+        admissions={}
+        for admission in db_admissions:
+            admissions[admission.student_id] = admission
+
         if self.request.method == 'POST':
             if not self.request.permissionInfo.has_permission('enter_points'):
                 return HTTPForbidden('Sie haben keine Rechte um Punkte einzutragen!')
             for student in students:
+                certificate_parameter = 'medical_certificate-{0}'.format(student.student_id)
+                if exam.medical_certificate and certificate_parameter in self.request.POST:
+                    admissions[student.student_id].medical_certificate = self.valueToBool(self.request.POST[certificate_parameter])
```

Comment:
Please split this in multiple lines and remove the function `valueToBool`. Just implement what valueToBool does as cases.

Reply:
```suggestion
                    medical_certificate_value = self.request.POST[certificate_parameter]
                    if medical_certificate_value == '1':
                        admissions[student.student_id].medical_certificate = True
                    elif medical_certificate_value == '0':
                        admissions[student.student_id].medical_certificate = False
                    else:
                        admissions[student.student_id].medical_certificate = None    
```

Would you prefer something like this?

Reply:
seems good. Please add it. I don't have access to the forked repository yet.